configure.ac

```diff
@@ -17,14 +17,10 @@ AC_SUBST([MAKEINFO])
 AM_INIT_AUTOMAKE([-Wall -Wno-portability dist-bzip2 dist-zip subdir-objects])
 
 AC_CONFIG_HEADERS([config.h])
-AH_BOTTOM([
-#include <helper/system.h>
-#include <helper/types.h>
-#include <helper/replacements.h>
-])
```

Comment:
Note: This is the change that required me to touch a bazillion .h files.

This change was required because otherwise compiling gnulib files gave me errors about not finding .h files. I couldn't see how to change things in the Makefiles to add the appropriate -I there without manually editing the auto-generated Makefile inside gnulib. These lines feel like a hack regardless, so removing them should be an improvement.

Reply:
@timsifive  - Hi Tim, it seems this change has further ripple-effect in other files. E.g. in usb_common.{c,h}, as reported here: https://github.com/riscv/riscv-openocd/issues/589. The issue can be reproduced with `./configure --enable-jlink [...]`. There may be more.

I wonder if there is a different solution that would allow to have gnulib and at the same time less impact on the existing code/headers. Lower amount of changes would also make the upstream acceptance easier.